### PR TITLE
fix bug celery.utils.time

### PR DESCRIPTION
**Pull request by sonpython**
fix bug https://github.com/celery/django-celery/issues/496
http://docs.celeryproject.org/en/latest/whatsnew-4.0.html#changes-to-internal-api

### CODE REVIEW

File: djcelery/managers.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 from django.db.models.query import QuerySet
 from django.conf import settings
 
-from celery.utils.timeutils import maybe_timedelta
+from celery.utils.time import maybe_timedelta
```
Comment (auvipy) [inline djcelery/managers.py]: how do you keep 3 and 4 compat? what about a try except block?

**Comment by thedrow** [comment]
Thank you for your contribution but the issue is already fixed.

**Comment by thedrow** [comment]
@auvipy Do you mind releasing a new version of django-celery?

**Comment by loisaidasam** [comment]
+1, I could also use an updated/fixed version 👍

**Comment by auvipy** [comment]
@thedrow only ask ken do that

**Pull request closed without merging**